application-ckeditor-ui/src/main/resources/CKEditor/EditSheet.xml

```diff
@@ -304,6 +309,7 @@ define('xwiki-ckeditor', [
       }
     }
 
+    var isTemporaryAttachmentSupported = $(element).attr('data-temporary-attachment-supported') || false;
```

Comment:
Do you still need this? I don't think so, since the config is now done in ``ConfigSheet``. No change is required to ``EditSheet`` I think.

Reply:
Right